client/asset/dash/dash_external_fee.go

```diff
@@ -6,60 +9,75 @@ import (
 	"errors"
 	"io"
 	"net/http"
+	"sync"
 	"time"
 
 	"decred.org/dcrdex/dex"
 )
 
 // See Also: https://blockchair.com/api/docs
 
+const (
+	minCacheTime       = 3 * time.Second
+	mainnetFeeStatsAPI = "https://api.blockchair.com/dash/stats"
+)
+
+var (
+	statsCache = blockchairStatsCache{
+		sync.Mutex{},
+		time.Now(),
+		blockchairStatsData{uint64(1)}}
+)
```

Comment:
I guess this works but normally we avoid global variables that are written to after initiation. To keep current patterns, you could create the cache in `newWallet` and it would be local to that created wallet. I think that would probably be best.

Reply:
Ok sure ...

Reply:
_I have explored this external fee implementation at length and these are my thoughts:_

1. We already have a working `estimatesmartfee` rpc. So the external estimator never really gets called
2. In case (1) we could now perhaps use the newer FeeRater.FeeRate, maybe..
3. The clone config functions are things like deserializers, serializers, decoders and do not normally have state
4. Even if this particular web function were called it is limited in requests served without payment. 1440/day (hence the cache)
5. I have never seen any value returned but 1 sat/byte. Maybe the server is hard coded ;-)

I can make it all work but it would require hacking
For now I will remove the external fee function in this PR